editor/src/clj/editor/code/view.clj

```diff
@@ -251,7 +258,8 @@
               (.fillRoundRect gc (.x r) (.y r) (.w r) (.h r) 5.0 5.0))
       :range (doseq [^Rect r rects]
                (.fillRect gc (.x r) (.y r) (.w r) (.h r)))
-      :underline nil)))
+      :underline nil
+      :diagnostic nil)))
```

Comment:
Could this be something more generic? `:squiggle` or `:issue`, perhaps? The `CursorRangeDrawInfo` `type` is typically associated with the type of visual element, separate from the `CursorRange` `type`.

Reply:
Good point, I'll make it a `:squiggle`